contracts/bridge/Bridge.sol

```diff
@@ -83,74 +85,169 @@ contract Bridge is IERC20BridgeReceiver, IERC721BridgeReceiver, Ownable, BridgeF
         uint64 handleNonce
     );
 
-    // start allows the value transfer request.
-    function start() external onlyOwner {
-        isRunning = true;
+    modifier onlySigners()
+    {
+        require(msg.sender == owner() || signers[msg.sender]);
+        _;
     }
 
-    // stop prevent the value transfer request.
-    function stop() external onlyOwner {
-        isRunning = false;
+    // start allows the value transfer request.
+    function start(bool _status, uint64 _requestNonce)
+        external
+        onlySigners
+    {
+        onlySequentialNonce(TransactionType.Governance, _requestNonce);
```

Comment:
Could you change this function to a modifier?

Reply:
Modifiers are compile-time source code roll-ups. I prefer a function calls to modifiers if it is called many times. In my case, modifiers causes a failure while deploying bridges.